app/utilities/UserMatchingSessionUtil.scala

```diff
@@ -50,6 +50,20 @@ object UserMatchingSessionUtil {
         dobY,
         nino
       )
+
+    def clearUserDetailsExpectName(implicit request: Request[AnyContent]): Result =
```

Comment:
```suggestion
    def clearUserDetailsExcectName(implicit request: Request[AnyContent]): Result =
```
Is this supposed to be Except rather than Expect?